llvm/lib/Transforms/IPO/MemProfContextDisambiguation.cpp

```diff
@@ -95,6 +95,7 @@ STATISTIC(NewMergedNodes, "Number of new nodes created during merging");
 STATISTIC(NonNewMergedNodes, "Number of non new nodes used during merging");
 STATISTIC(MissingAllocForContextId,
           "Number of missing alloc nodes for context ids");
+STATISTIC(SkippedCallsCloning, "Number of calls skipped during cloning");
```

Comment:
I wonder if this could be more specific to the reason it was skipped? On the other hand, is this case the only place where we may skip calls during cloning?

Reply:
Maybe the ICP case is good to cover with this statistic too? Then we can keep the name and description as is. 

Not a strong opinion, feel free to just update the description as you suggested as an alternative.